packages/apollo-engine-reporting/src/schemaReporter.ts

```diff
@@ -0,0 +1,161 @@
+// Fields required for the trial related emails
+import gql from 'graphql-tag';
+import {
+  AutoregReportServerInfo,
+  ReportServerInfoVariables,
+  EdgeServerInfo,
+} from './reportingOperationTypes';
+import { DocumentNode, execute, makePromise, Observable } from 'apollo-link';
+import { FetchResult } from 'apollo-link/lib/types';
+import { HttpLink } from 'apollo-link-http';
+import { fetch } from 'apollo-server-env';
+import { Logger } from 'apollo-server-types';
+
+const reportServerInfoGql = gql`
```

Comment:
Can we also port the unit tests?

Reply:
Ported over the relevant tests. They are in `plugin.test.ts` and `agent.test.ts`